app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    compileSdkVersion 25
```

Comment:
Any reason we're lowering the compile and target versions?

Reply:
I had tried to add design support library to exiting sdk, didn't work. So, followed the Twitter app guide, and changed according.